src/main/java/ch/njol/skript/effects/EffItemLifetime.java

```diff
@@ -0,0 +1,57 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Unlimited Item Lifetime")
+@Description("Makes a dropped item have unlimited lifetime, meaning it won't despawn from vanilla Minecraft timer.")
+@Examples({
+	"enabled unlimited lifetime of all dropped items"
+})
+@Since("INSERT VERSION")
+public class EffItemLifetime extends Effect {
+
+	static {
+		Skript.registerEffect(EffItemLifetime.class,
+			"enable unlimited lifetime for %itementities%",
+			"make lifetime unlimited for %itementities%",
+			"make [the] %itementities% lifetime unlimited",
```

Comment:
```suggestion
```
these dont really work as well as the first one. the last one is also not correct grammatically

Reply:
Hmmm, opposite opinions between you and Pickle.